packages/neuron-ui/src/services/UILayer.ts

```diff
@@ -140,13 +142,22 @@ export const walletsCall = instantiateMethodCall(wallets) as {
   importKeystore: (params: { name: string; keystore: string; password: string }) => void
   importMnemonic: (params: { name: string; mnemonic: string; password: string }) => void
   update: (params: { id: string; name?: string; password: string; newPassword?: string }) => void
-  delete: (params: { id: string; password: string }) => void
   export: (id: string) => void
   getActive: () => void
   activate: (id: string) => void
   backup: (id: string) => void
 }
 
+export const deleteWallet = (id: string, password: string, handleResult: any) => {
```

Comment:
Why move `delete` to an independent channel?

Reply:
It should validate password before deleting wallet.If password is wrong,I should notify user in the dialog.It likes sync method.

Reply:
In this case, pass id and password to the neuron-wallet together, and keep the dialog open.

If the password is correct, delete the wallet and broadcast to neuron-ui. neuron-ui hdies the dialog.

If the password is incorrect, send error message to neuron-ui, and neuron-ui render it on the dialog.

Reply:
fixed